prime-angular-frontend/src/app/modules/adjudication/shared/components/maintenance-container/maintenance-container.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="container-fluid">
+  <div class="row">
+    <div class="col-md-6 col-sm-12">
+      <mat-card class="has-hover">
+        <mat-card-header>
+          <div mat-card-avatar>
+            <mat-icon color="primary">email</mat-icon>
+          </div>
+          <mat-card-title>Notification Emails</mat-card-title>
+        </mat-card-header>
+        <mat-card-content>
+          View the notification emails.
+        </mat-card-content>
+        <mat-card-actions>
+          <button mat-stroked-button
+                  type="button"
+                  color="primary"
+                  (click)="onRoute(AdjudicationRoutes.NOTIFICATION_EMAILS)">
+            View
+          </button>
```

Comment:
Was trying as a non-superadmin role, could see the View button on frontend but got backend 403 forbidden error after clicking view button. Feels like need to only enable the button for superadmin.

Reply:
Have a note i forgot to enable the back end for all admin types have it restricted. so changes coming down